src/postgres/utils/sql.ts

```diff
@@ -161,8 +161,7 @@ namespace sql {
   function escapeSqlIdentifier (str: string): string {
     let escaped = '"'
 
-    for (let i = 0; i < str.length; i++) {
-      let c = str[i]
+    for (const c of str) {
       if (c === '"') escaped += c + c
       else escaped += c
     }
```

Comment:
Would a simple `escaped += str.replace(/"/g, '""')` not have been sufficient (and more performant) here? Is there a situation where it falls down?